.github/workflows/scripts/run_unit_tests.sh

```diff
@@ -14,3 +14,9 @@ if [ -n "${COVERALLS_TOKEN}" ]; then
     "$(go env GOPATH)"/bin/goveralls -coverprofile="${COVERPROFILE}" \
             -service=github
 fi
+
+# re-run with race detector
```

Comment:
I think I'd prefer that these race tests are run in parallel to the normal tests by the CI/CD pipeline to hopefully not extend our pipeline run time.

Reply:
Should I do that as a separate script? I'm not sure the best way to do that and this was the easiest way to get started running them at all.

Reply:
sorted, I think